editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2039,10 +2051,22 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("ColorConstant", "Color", "Variables", "VisualShaderNodeColorConstant", TTR("Color constant."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 	add_options.push_back(AddOption("ColorUniform", "Color", "Variables", "VisualShaderNodeColorUniform", TTR("Color uniform."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 
-	// BOOLEAN
+	// CONDITIONAL
 
+	add_options.push_back(AddOption("Equal", "Conditional", "Functions", "VisualShaderNodeCompare", TTR(vformat("Returns the boolean result of the contains the result of %s comparison between two parameters.", "Equal (==)")), VisualShaderNodeCompare::FUNC_EQUAL, VisualShaderNode::PORT_TYPE_BOOLEAN));
```

Comment:
It should be the other way around, `vformat(TTR("Returns the boolean result of the contains the result of %s comparison between two parameters."), TTR("Equal (==)"))`. `TTR` can and should only be used on constant strings, it's extracted by parsing the source code, not executing it.

You could also extract the string in a variable to avoid code duplication:
```cpp
const String &compare_func_desc = TTR("Returns the boolean result of the contains the result of %s comparison between two parameters.");
add_options.push_back(AddOption("Equal", "Conditional", "Functions", "VisualShaderNodeCompare", vformat(compare_func_desc, TTR("Equal (==)"))), VisualShaderNodeCompare::FUNC_EQUAL, VisualShaderNode::PORT_TYPE_BOOLEAN));
...
```

Reply:
True.. fixed